pdany1116/todo_api
# Add rubocop

#### Description

Rubocop linter was added.

#### [Task](https://trello.com/c/j2IfeG25/16-add-rubocop)

#### Related PRs




## Steps to Test or Reproduce

1. Checkout on this branch.
2. Run `rubocop`.
3. No offenses should be detected.

## Risks
